ci-scripts/schedule_k8s_deploy.sh

```diff
@@ -9,34 +9,46 @@ branch=${TEST_BRANCH-main}
 network=${1-dev}
 environment=ceramic-v4-${network}
 
+if [[ "$network" == "dev" || "$network" == "qa" ]]; then
+    ceramic_image_name="ceramicnetwork/js-ceramic:develop"
+elif [[ "$network" == "tnet" ]]; then
+    ceramic_image_name="ceramicnetwork/js-ceramic:release-candidate"
+elif [[ "$network" == "prod" ]]; then
+    ceramic_image_name="ceramicnetwork/js-ceramic:latest"
+else
+    echo "Invalid network value: $network"
+    exit 1
+fi
+
 docker run --rm -i \
   -e "AWS_REGION=$AWS_REGION" \
   -e "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID" \
   -e "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" \
   -v ~/.aws:/root/.aws \
   -v "$PWD":/aws \
   amazon/aws-cli dynamodb put-item --table-name "ceramic-$network-ops" --item \
-  "{                                                        \
-    \"id\":     {\"S\": \"$id\"},                           \
-    \"job\":    {\"S\": \"$job_id\"},                       \
-    \"ts\":     {\"N\": \"$now\"},                          \
-    \"ttl\":    {\"N\": \"$ttl\"},                          \
-    \"stage\":  {\"S\": \"queued\"},                        \
-    \"type\":   {\"S\": \"workflow\"},                      \
-    \"params\": {                                           \
-      \"M\": {                                              \
-        \"name\":     {\"S\": \"Deploy CERAMIC ONE\"},      \
-        \"org\":      {\"S\": \"3box\"},                    \
-        \"repo\":     {\"S\": \"ceramic-infra\"},           \
-        \"ref\":      {\"S\": \"$branch\"},                 \
-        \"workflow\": {\"S\": \"update_image.yml\"},        \
-        \"labels\":   {\"L\": [{\"S\": \"deploy\"}]},       \
-        \"inputs\":   {                                     \
-          \"M\": {                                          \
-            \"ceramic_one_image\": {\"S\": \"$image\"},     \
-            \"environment\": {\"S\": \"$environment\"}      \
-          }                                                 \
-        }                                                   \
-      }                                                     \
-    }                                                       \
+  "{                                                            \
+    \"id\":     {\"S\": \"$id\"},                               \
+    \"job\":    {\"S\": \"$job_id\"},                           \
+    \"ts\":     {\"N\": \"$now\"},                              \
+    \"ttl\":    {\"N\": \"$ttl\"},                              \
+    \"stage\":  {\"S\": \"queued\"},                            \
+    \"type\":   {\"S\": \"workflow\"},                          \
+    \"params\": {                                               \
+      \"M\": {                                                  \
+        \"name\":     {\"S\": \"Deploy CERAMIC ONE\"},          \
```

Comment:
Can we update the string to something like `Deploy k8s dev CERAMIC ONE`? We don't currently display which env is being deployed to 😬
```suggestion
        \"name\":     {\"S\": \"Deploy k8s $network CERAMIC ONE\"},          \
```

Reply:
You'll have to update the indentation of the `\`s...

Reply:
And feel free to setup the display string however is most clear as long as the network name is included.